src/helpers/validation.ts

```diff
@@ -1,10 +1,32 @@
-import Ajv from 'ajv';
+import Ajv, { ErrorObject } from 'ajv';
 import { validateAndParseAddress } from 'starknet';
 import { isAddress } from '@ethersproject/address';
 import { parseUnits } from '@ethersproject/units';
 import { Zero, MinInt256, MaxInt256, MaxUint256 } from '@ethersproject/constants';
 import { BigNumber } from '@ethersproject/bignumber';
 
+function getErrorMessage(errorObject: ErrorObject): string {
+  if (!errorObject.message) return 'Invalid field.';
+
+  if (errorObject.keyword === 'format') {
+    switch (errorObject.params.format) {
+      case 'address':
+        return 'Must be a valid address.';
+      case 'uint256':
+      case 'int256':
+        return `Must be a valid ${errorObject.params.format} value.`;
+      case 'ethValue':
+        return 'Must be a valid Ethereum value.';
```

Comment:
This might be confusing, I'd rather say something like:
```suggestion
        return 'Must be a number.';
```